etools-datepicker-button.html

```diff
@@ -194,6 +197,11 @@
               });
         }
       }
+
+      _minMaxDateChanged(minDate, maxDate) {
+        this.datePicker.minDate = minDate;
+        this.datePicker.maxDate = maxDate;
+      }
```

Comment:
I'm a bit concerned about this, we've had in the past cases where change events were fired before `connectedCallback()`, and in this case `this.datePicker` would be undefined. Should we consider checking for undefined?